components/org.wso2.carbon.identity.sso.saml.ui/src/main/java/org/wso2/carbon/identity/sso/saml/ui/client/SamlSPMetadataUploadExecutor.java

```diff
@@ -50,7 +50,7 @@ public boolean execute(HttpServletRequest httpServletRequest, HttpServletRespons
         String webContext = (String) httpServletRequest.getAttribute(CarbonConstants.WEB_CONTEXT);
         String serverURL = (String) httpServletRequest.getAttribute(CarbonConstants.SERVER_URL);
         String cookie = (String) httpServletRequest.getAttribute(ServerConstants.ADMIN_SERVICE_COOKIE);
-        String spName = httpServletRequest.getParameter("application-sp-name");
+        String spName = getFormFieldsMap().get("application-sp-name").get(0);
```

Comment:
Let's define a constant for 'application-sp-name'

Reply:
defined a constant named `APPLICATION_SP_NAME_KEY`.